.github/workflows/ci.yaml

```diff
@@ -113,11 +113,27 @@ jobs:
       - uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5.1.0
         with:
           go-version-file: "${{ github.workspace }}/go.mod"
+      - name: Set up Helm
```

Comment:
why do we need this?

Reply:
`make doc` depends on `make manifests` which uses helm to generate the files